### PR TITLE
Improve Page Builder Interface accessibility

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Resolve https://github.com/siteorigin/siteorigin-panels/issues/765

Status (pending approval):
- [x] Navigating page builder rows, and columns
- [x] Add Widgets view
- [x] Layouts view
- [x] History
- [x] Live Editor
- [x] Row/Widget interface navigation
- [x] Widget contents navigation https://github.com/siteorigin/so-widgets-bundle/pull/1206

Note:
- There's no way to reorder widgets or rows. This needs further investigation and will be implemented in a future PR.

### CODE REVIEW

File: js/siteorigin-panels/view/builder.js [hunk 1]
```diff
@@ -247,9 +249,13 @@ module.exports = Backbone.View.extend( {
 
 		// Switch back to the standard editor
 		if ( this.supports( 'revertToEditor' ) ) {
-			metabox.find( '.so-switch-to-standard' ).click( function ( e ) {
+			metabox.find( '.so-switch-to-standard' ).on( 'click keyup', function ( e ) {
```
Comment (Misplon) [inline js/siteorigin-panels/view/builder.js:252]: Perhaps changed to `function( e )`

File: js/siteorigin-panels/helpers/accessibility.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+var $ = jQuery;
+
+module.exports = {
+	/**
+	 * Trigger click on valid enter key press.
+	 */
+	triggerClickOnEnter: function( e ) {
+		if ( e.which == 13 ) {
+			$(e.target).trigger('click');
```
Comment (Misplon) [inline js/siteorigin-panels/helpers/accessibility.js:9]: Perhaps `$( e.target ).trigger( 'click' );`

**Comment by rockstarrossi** [comment]
Yes

**Comment by AlexGStapleton** [comment]
@Misplon The last commit will handle the widget and layouts views.

Some thoughts:

Can you think of a way to trigger the context menu? We could use a button press, but how would we communicate that? It really needs to be obvious.

How do you want to handle the live editor? I honestly can't think of a way for us to reliabily get it working as we can't always rely on the theme to handle things in a consistent manner.

**Comment by Misplon** [comment]
Discussion was had via Slack.

**Review by Misplon — APPROVED** [review]

**Review by gregpriday — APPROVED** [review]

**Pull request merged**